cvs diff nsPluginsDirWin.cpp (in directory E:\Projects\mozilla\modules\plugin\nglsrc\)
Index: nsPluginsDirWin.cpp
===================================================================
RCS file: /cvsroot/mozilla/modules/plugin/nglsrc/nsPluginsDirWin.cpp,v
retrieving revision 1.18
diff -r1.18 nsPluginsDirWin.cpp
34a35
> #include "prprf.h"
38a40,42
> #include "nsSpecialSystemDirectory.h"
> 
> 
126a131,132
>   // Use the Moz_BinDirectory
>   nsSpecialSystemDirectory plugDir(nsSpecialSystemDirectory::Moz_BinDirectory);
130,150c136,142
<     // look for a plugin folder that exists in the same directory as our executable
<     if (::GetModuleFileName(NULL, path, sizeof(path)) > 0) 
<     { 
<       pathlen = PL_strlen(path) - 1; 
< 
<       while (pathlen > 0) 
<       { 
<         if (path[pathlen] == '\\') 
<           break; 
< 
<         pathlen--; 
<       }
< 
<       if (pathlen > 0) 
<       { 
<         PL_strcpy(&path[pathlen + 1], "plugins"); 
< 
< 	      allocPath = path;
< 	      *(nsFileSpec*)this = allocPath;
<       } 
<     } 
---
>     // look for a plugin folder that exists in the same directory as 
>     // the mozilla bin directory
> 
>     plugDir += "plugins";
>     *(nsFileSpec*)this = plugDir;
>     PR_snprintf(path, 2000, "%s", (const char *) plugDir);
>     

*****CVS exited normally with code 1*****

